plugins/thunderbolt/thunderbolt.conf

```diff
@@ -4,3 +4,6 @@
 # It's important that all backports from this kernel have been
 # made if using an older kernel
 MinimumKernelVersion=4.13.0
+
+# Forces delaying activation until shutdown/logout/reboot
+DelayedActivation=false
```

Comment:
why a config option?

Reply:
ChromeOS will want to turn this on for everything with a guarantee that they always run activation on logout/shutdown/reboot.  I didn't think this made sense for generically doing in all distros.  

If we want to shift fwupd to behave this way, we can drop the config option (and some logic).  See the matrix of situations to support in the commit message for background.